src/main/kotlin/no/nav/syfo/oppfolgingstilfelle/database/OppfolgingstilfelleArbeidstakerQuery.kt

```diff
@@ -57,15 +59,15 @@ const val queryGetOppfolgingstilfelleArbeidstaker =
         ORDER BY referanse_tilfelle_bit_inntruffet DESC;
     """
 
-fun DatabaseInterface.getOppfolgingstilfelleArbeidstaker(
+fun Connection.getOppfolgingstilfelleArbeidstakerList(
```

Comment:
Synes kanskje det var bedre å ha DatabaseInterface her, for sånn den er blitt nå så har den som sideeffekt å lukke connection.